src/frontend/src/App.js

```diff
@@ -1,26 +1,28 @@
 import React from "react";
-import page404 from "./components/404";
 import logo from "./logo.svg";
 import "./App.css";
+import page404 from "./components/404";
+import SignUp from "./components/signup";
 
-function App() {
-	return (
-		<div className="App">
-			<header className="App-header">
-				<img src={logo} className="App-logo" alt="logo" />
-				<p>
-					Edit <code>src/App.js</code> and save to reload.
-				</p>
-				<a
-					className="App-link"
-					href="https://reactjs.org"
-					target="_blank"
-					rel="noopener noreferrer">
-					Learn React
-				</a>
-			</header>
-		</div>
-	);
-}
+// function App() {
+// 	return (
+// 		<div className="App">
+// 			<header className="App-header">
+// 				<img src={logo} className="App-logo" alt="logo" />
+// 				<p>
+// 					Edit <code>src/App.js</code> and save to reload.
+// 				</p>
+// 				<a
+// 					className="App-link"
+// 					href="https://reactjs.org"
+// 					target="_blank"
+// 					rel="noopener noreferrer">
+// 					Learn React
+// 				</a>
+// 			</header>
+// 		</div>
+// 	);
+// }
```

Comment:
Why commenting this out?
If not required delete it.

Reply:
sure